### PR TITLE
Update `register_service` script and add some tests

### LINKED ISSUE #491: Update register-service script for new contracts

### PR DESCRIPTION
**Pull request by palango**
There should be an CLI test as well, but this requires so much mocking that I didn't dare to do it now.

Part of #452 
Fixes #491

### CODE REVIEW

File: src/raiden_libs/register_service.py [hunk 1]
```diff
@@ -20,6 +20,26 @@
...
+    web3: Web3, service_address: Address, function_call: ContractFunction, task_name: str
+) -> None:
+    log.info(f"Starting: {task_name}")
+    transaction_hash = function_call.transact({"from": service_address})
+    transaction_receipt = web3.eth.waitForTransactionReceipt(transaction_hash)
+    was_successful = transaction_receipt["status"] == 1
+
+    if not was_successful:
+        log.error(
+            f"Failed: {task_name}\nPlease check that you account is funded.",
```
Comment (Dominik1999) [inline src/raiden_libs/register_service.py]: you**r** account is funded

But maybe even -> "Please check that you have enough funds" ?

File: src/raiden_libs/register_service.py [hunk 2]
```diff
@@ -65,48 +82,86 @@ def main(
...
-            log.info("Updated account balance", balance=account_balance, desired_deposit=deposit)
 
-        # Approve token transfer
+def deposit_to_registry(
+    web3: Web3,
+    service_registry_contract: Contract,
+    user_deposit_contract: Contract,
+    service_address: Address,
+) -> None:
+    log.info("Address not registered in ServiceRegistry")
```
Comment (Dominik1999) [inline src/raiden_libs/register_service.py]: how do you know that? Do you check for duplicates somewhere?

File: src/raiden_libs/register_service.py [hunk 3]
```diff
@@ -65,48 +82,86 @@ def main(
 
     # Register if not yet done
     if current_deposit <= 0:
```
Comment (Dominik1999) [inline src/raiden_libs/register_service.py]: is that the stake of the service? then maybe call it stake?

Comment (palango) [reply src/raiden_libs/register_service.py]: Good point, I wonder how that actually worked. There is no field `deposits` any more?!

File: src/raiden_libs/register_service.py (hunk 2 above)
Comment (palango) [reply src/raiden_libs/register_service.py]: `deposit_to_registry` is only called if the check you commented on above is false.

**Pull request merged**